src/raftstore/store/fsm/peer.rs

```diff
@@ -1466,6 +1489,7 @@ impl<'a, T: Transport, C: PdClient> PeerFsmDelegate<'a, T, C> {
                 }
                 self.fsm.peer.recent_conf_change_time = now;
                 self.fsm.peer.insert_peer_cache(peer);
+                // Update group config
```

Comment:
Do you mean we also need to know the peer's location?

Reply:
Sry, I forgot to remove this comment. The config will be updated in L1521